### PR TITLE
New package: gstreamerproject.gstreamer.devel version 1.22.7

### LINKED ISSUE #107954: [Package Request]: gstreamer development & runtime
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://gstreamer.freedesktop.org/download/
Publisher: gstreamer
Package Name: gstreamer.runtime / gstreamer.development

right now only the runtimes are provided at `gstreamer.gstreamer`. "For development, you will want to install both packages" per freedesktop themselves at https://gstreamer.freedesktop.org/download/ The packages should be split into `gstreamer.gstreamer.development` and `gstreamer.gstreamer.runtime` so devs know they need both.

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve #107954

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/130584)

### CODE REVIEW

**Comment by Dragon1573** [comment]
Pull request #130582 is successfully merged. Please manually restart the validation pipeline.

**Comment by stephengillie** [comment]
image

File: manifests/g/gstreamerproject/gstreamer/devel/1.22.7/gstreamerproject.gstreamer.devel.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: gstreamerproject.gstreamer.devel
+PackageVersion: 1.22.7
+InstallerLocale: en-US
+InstallerType: wix
+Dependencies:
+  PackageDependencies:
```
Comment (stephengillie) [inline manifests/g/gstreamerproject/gstreamer/devel/1.22.7/gstreamerproject.gstreamer.devel.installer.yaml:9]:
```suggestion
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by Dragon1573** [comment]
@Trenly @stephengillie - This package can be successfully installed on GitHub-hosted Runner. I think it's a false-positive report of https://github.com/microsoft/winget-pkgs/labels/URL-Validation-Error . I can't reproduce this behaviour.

- Check [logs_6.zip](https://github.com/microsoft/winget-pkgs/files/13815659/logs_6.zip) for more informations.

**Comment by stephengillie** [comment]
Hi @Dragon1573,

These are probably data center blocks, to prevent automated downloads of the website. Unfortunately, I can't easily waiver or get around these errors.

**Comment by Dragon1573** [comment]
Em ... :confused:

Okay, I'll rrgard it as a "stable reproduce-able" problem. This blocks us from downloading installer file in the data center.

I'll close this pull request as "rejected". Thanks :heart:

**Comment by luzpaz** [comment]
Heads up, 1.22.8 has been released (https://gstreamer.freedesktop.org/releases/1.22/#1.22.8)

**Pull request closed without merging**